guide/how-it-works/silent-payments.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: guide
+title: Silent payments
+description: How silent payments impact and improve bitcoin UX
+nav_order: 11
+parent: How it works
+permalink: /guide/how-it-works/silent-payments/
+redirect_from:
```

Comment:
You can delete the redirect lines. Those are only if a page URL is changed, to forward old links to the new one.

Reply:
Fixed in https://github.com/BitcoinDesign/Guide/pull/1109/commits/ac341d1d28599f83bd4aaf955467f51d00caadf6